library/MedEx/API.php

```diff
@@ -2913,136 +2913,135 @@ public function MedExBank($stage)
+                                                    <i id="email_check" name="email_check" class="top_right_corner nodisplay text-success fa fa-check"></i>
+                                                    <input type="text" data-rule-email="true" class="form-control" id="new_email" name="new_email" value="<?php echo attr($GLOBALS['user_data']['email']); ?>" placeholder="<?php echo xla('your email address'); ?>" required>
+                                                    <div class="signup_help nodisplay" id="email_help" name="email_help"><?php echo xlt('Please provide a valid e-mail address to proceed'); ?>...</div>
+                                                </div>
+                                            </div>
+                                            <div class="divTableRow">
+                                                <div class="divTableCell divTableHeading">
+                                                    <?php echo xlt('Password'); ?>
+                                                </div>
+                                                <div class="divTableCell"><i id="pwd_check" name="pwd_check" class="top_right_corner nodisplay text-success fa fa-check"></i>
+                                                    <i class="fa top_right_corner fa-question" id="pwd_ico_help" aria-hidden="true" onclick="$('#pwd_help').toggleClass('nodisplay');"></i>
+                                                    <input type="password" placeholder="<?php xla('Password'); ?>" id="new_password" name="new_password" class="form-control" required>
+                                                    <div id="pwd_help" class="nodisplay signup_help"><?php echo xlt('Secure Password Required') . ": " . xlt('8-12 characters long, including at least one upper case letter, one lower case letter, one number, one special character and no common strings'); ?>...</div>
+                                                </div>
+                                            </div>
+                                            <div class="divTableRow">
+                                                <div class="divTableCell divTableHeading">
+                                                    <?php echo xlt('Repeat'); ?>
+                                                </div>
+                                                <div class="divTableCell"><i id="pwd_rcheck" name="pwd_rcheck" class="top_right_corner nodisplay text-success fa fa-check"></i>
+                                                    <input type="password" placeholder="<?php echo xla('Repeat password'); ?>" id="new_rpassword" name="new_rpassword" class="form-control" required>
+                                                    <div id="pwd_rhelp" class="nodisplay signup_help" style=""><?php echo xlt('Passwords do not match.'); ?></div>
+                                                </div>
+                                            </div>
```

Comment:
Hi @bradymiller @tywrenn  and @ophthal

I would like to plan to change the table to the Bootstrap form-group style, What do you think? let me know your thought.